module/observationSubmissions/helper.js

```diff
@@ -407,10 +408,11 @@ module.exports = class ObservationSubmissionsHelper {
     * @param {String} - entityId
     * @param {String} - solutionId
     * @param {String} - observationId
+    * @param {String} - userId
     * @returns {Object} - list of submissions
     */
 
-   static list(entityId,observationId) {
+   static list(entityId,observationId,userId) {
```

Comment:
@VISHNUDAS-tunerlabs why is additional userId required in the argument if not used in the function?